doc/content/installation.md

```diff
@@ -4,17 +4,17 @@ fillname requires Python 3.8+. We recommend version 3.10.
 
 You can check a successful installation by running
 
-```shell
-fillname -h
+```console
+$ fillname -h
```

Comment:
Hi @susuhahnml, why did you choose to use console and dollar sign here? I actually like the version in the README better.
The dollar sign makes it harder to just copy the code into the terminal.

Reply:
I sometimes use the `$` sign to distinguish code and output. If there is just code, then there is no need for the `$`.

Reply:
In the documentation the copy button will ignore this sign. And make a nicer format.

Reply:
If you don't want to compile the documentation, just check out the clinguin or clingraph read the docs as an example

Reply:
I just checked the clinguin example, and it did not work for me. The dollar sign was also copied. :cry:

https://clinguin.readthedocs.io/en/latest/clinguin/use.html

```console
$ clinguin server --domain-files examples/angular/sudoku/instance.lp examples/angular/sudoku/encoding.lp --ui-files examples/angular/sudoku/ui.lp
```

Reply:
Hey! Sorry, the clinguin docu aparently is not updated. Check the clingraph one: https://clingraph.readthedocs.io/en/latest/clingraph/installation.html 

With this package the prompt should not be copied.

Reply:
Is that why you added the complicated regexp? I admit I did not try to understand what is happening there. It somehow seems a lot of effort just to get a dollar sign.

Reply:
Yes. Because it covers other prompts. We can simplify it and sick with one, `>` or `$`

Reply:
This comes back to Sven's comment. Why have it at all? One could simply write the commands without a prefix and does not even need a complicated copy button. I'll leave it for you to decide. Just my two cents.

Reply:
Personally I like to have the prompt. I think is more intuitive.

Reply:
Yes, that worked :+1: